src/modules/data/constants/MQAcceptedPatterns.ts

```diff
@@ -0,0 +1,20 @@
+// CAREFUL modifying this file! The values have to match what the Aggregator sends exactly or we will get
```

Comment:
Maybe put this in a shared repo so they can be synced?

Reply:
Will create an issue to do this, out of scope for this PR